gui/tasks/distribution.js

```diff
@@ -328,6 +359,27 @@ function root(relativePath) {
   return path.join(path.resolve(__dirname, '../../'), relativePath);
 }
 
+function getLinuxTargetArch() {
+  if (targets) {
+    if (targets === 'aarch64-unknown-linux-gnu') {
+      return 'arm64';
+    }
+    throw new Error(`Invalid or unknown target (only one may be specified)`);
+  }
+  // Use host architecture.
+  return undefined;
+}
+
+function getLinuxTargetSubdir() {
```

Comment:
For macOS we always provide a target when building the rust binaries resulting in them being placed in a target directory. This makes it possible to always just use the target triple in the path, e.g. `{ from: distAssets(path.join('${env.TARGET_TRIPLE}', 'mullvad')), to: '.' }`. What do you think about doing the same here?

This is the call in `build.sh` when no target is provided:
```
    if [[ "$(uname -s)" == "Darwin" ]]; then
        # Provide target for non-universal macOS builds to use the same output location as for
        # universal builds
        build "$ENV_TARGET"
    else
        build
    fi
```

Reply:
After discussing this in the office David conviced me that this approach doesn't work due to file patterns not being supported in fpm arguments.